mendix/widgets-resources
# Update libraries and datasource attribute function

**In this PR I am:**
- Updating mendix lib and big.js
- Migrating calls for datasource attributes to `.get(objectitem)`

**Why?**
- Because the usage of `attribute(item)` was deprecated in 9.0.5 and it appears a warning for each property using it.

**How to test?**
- Manually install the typing library from 9.0.5 pipeline
- Test the affected widgets and check if they are still throwing warnings.

_Note: No new function was changed/added._
_Note 2: It requires a lib not released yet, so builds will fail._
